conans/client/tools/system_pm.py

```diff
@@ -54,6 +55,21 @@ def _create_tool(os_info):
         else:
             return NullTool()
 
+    def add_repositories(self, repositories, repo_keys=None, update=True):
```

Comment:
Personally, I would prefer the user calling N times to `add_repository` and avoid the complex parameters. I think typically they will add 1 or 2 max.